main.cpp

```diff
@@ -1 +1,215 @@
+        {
+            if(xCoord <= SIZE && xCoord > 0 && yCoord <= SIZE && yCoord > 0)
+            {
+                //std::cout << "Played on cell " + std::to_string((xCoord*10) + yCoord) << std::endl;
+                if(board[xCoord][yCoord] == 0)
+                {
+                    board[xCoord][yCoord] = isCrossPlaying ? -1 : 1;
+                    stayInLoop = false;
+                }
+                else
+                {
+                    std::cout << "You can't play on a non-empty cell. Please play again." << std::endl;
+                }
+            }
+            else
+            {
+                std::cout << "cell[0] = " + std::to_string(cell[0]) + " and cell[1] = " + std::to_string(cell[1]) << std::endl;
+                std::cout << "You can't enter coordinates below 0 or above the size of the playing board (" + std::to_string(SIZE) + ")." << std::endl;
+            }
+        }
+        else
+        {
+            std::cout << "Please enter only to numbers (x and y coordinate) ranging from 1 to " + std::to_string(SIZE) + "." << std::endl;
+        }
+    } while (stayInLoop);
+}
+
+/*
+ * Gère un tour de jeu pour une IA
+ */
+void AIPlay(bool isCrossPlaying)
+{
+
+}
+
+/*
+ * Parcours le tableau et détermine s'il y a un gagnant.
+ */
+uint8_t CheckForWinner()
+{
+    bool equality = true;
+    int8_t row = 0,column = 0,diagonal1 = 0, diagonal2 = 0;
+
+    for(auto & i : board)
+    {
+        for(uint8_t j=0;j<SIZE;j++)
+        {
+            if(i[j] == 0) equality = false;
+        }
+    }
+
+    if(!equality)
+    {
+        for(uint8_t i=0;i<SIZE;i++)
+        {
+            for(uint8_t j=0;j<SIZE;j++)
+            {
+                row += board[i][j];
+                column += board[j][i];
+                if(i == j)
+                {
+                    diagonal1 += board[i][j];
+                }
+                if(i == SIZE-j-1)
+                {
+                    diagonal2 += board[i][j];
+                }
+            }
+            if(row == SIZE || column == SIZE || diagonal1 == SIZE || diagonal2 == SIZE)
+            {
+                return 1;
+            }
+            else if (row == -SIZE || column == -SIZE || diagonal1 == -SIZE || diagonal2 == -SIZE)
+            {
+                return 2;
+            }
+
+            row = 0;
+            column = 0;
+        }
+    }
+    else
+    {
+        std::cout << "Nobody won..." << std::endl;
+        return 2;
+    }
+
+    return 0;
+}
+
+/*
+ * Gère le jeu complet
+ */
+void GameLoop()
+{
+    bool inGame = true;
+    bool isCrossPlaying = rand()%2;
+    bool playMode;
+
+    std::string playerChoice;
+    std::cout << "Choose play mode (1 = 2-player, 2 = player VS Human)" << std::endl;
+    std::cin >> playerChoice;
+    playerChoice == "1" ? playMode = false : playMode = true;
+
+    isCrossPlaying ? std::cout << "Cross starts to play as P1" << std::endl : std::cout << "Circle starts to play as P1" << std::endl;
```

Comment:
Same as above: you can give the result of the ternary operator directly to `cout`.